src/js/components/atoms/c-image/c-image.ts

```diff
@@ -38,23 +41,43 @@ export default class CImage extends Vue implements ICImageProps {
     quality!: number;
```

Comment:
Group decorators and internal props serperatly

Reply:
Hope you get those new :eyeglasses: soon ;-)